### ISSUE TITLE
Python 3.12 Installation error with lib-pod5 version conflicts

### ISSUE THREAD

**Issue by zg404** [nanoporetech/pod5-file-format#78]
Just attempted a fresh install within a new conda env using Python 3.12, and the pip install of pod5 did not work due to an "ERROR: ResolutionImpossible" with the lib-pod5 dependency. Downgrading to Python 3.11 was an easy fix, but I wanted to give this notice.

Here's the error:
```
ERROR: Cannot install pod5==0.0.43, pod5==0.1, pod5==0.1.10, pod5==0.1.11, pod5==0.1.12, pod5==0.1.13, pod5==0.1.15, pod5==0.1.16, pod5==0.1.19, pod5==0.1.20, pod5==0.1.21, pod5==0.1.4, pod5==0.1.5, pod5==0.2.0, pod5==0.2.2, pod5==0.2.3 and pod5==0.2.4 because these package versions have conflicting dependencies.

The conflict is caused by:
    pod5 0.2.4 depends on lib-pod5==0.2.4
    pod5 0.2.3 depends on lib-pod5==0.2.3
    pod5 0.2.2 depends on lib-pod5==0.2.2
    pod5 0.2.0 depends on lib-pod5==0.2.0
    pod5 0.1.21 depends on lib-pod5~=0.1.21dev0
    pod5 0.1.20 depends on lib-pod5~=0.1.20dev0
    pod5 0.1.19 depends on lib-pod5~=0.1.19dev0
    pod5 0.1.16 depends on lib-pod5~=0.1.15dev0
    pod5 0.1.15 depends on lib-pod5~=0.1.15dev0
    pod5 0.1.13 depends on lib-pod5~=0.1.10dev0
    pod5 0.1.12 depends on lib-pod5~=0.1.10dev0
    pod5 0.1.11 depends on lib-pod5~=0.1.10dev0
    pod5 0.1.10 depends on lib-pod5~=0.1.10dev0
    pod5 0.1.5 depends on lib-pod5~=0.1.5
    pod5 0.1.4 depends on lib-pod5~=0.1.4
    pod5 0.1 depends on lib-pod5~=0.1
    pod5 0.0.43 depends on lib-pod5~=0.0.43

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

**Comment by Psy-Fer** [comment]
You can see here
https://pypi.org/project/lib-pod5/#files
There is no build for python3.12, it only goes up to python3.11
ONT will have to build a release for python3.12

**Comment by zg404** [comment]
Gotcha, I wasn't sure if this belonged here or there. Like I said, downgrading to 3.11 was an immediate fix so should I close this?

**Comment by Psy-Fer** [comment]
Nah leave it. They need to add python3.12 to their release builds. This issue will remind them to do it. 😁

**Comment by jayhesselberth** [comment]
Bumping this, can you please release an update for Python 3.12?

**Comment by jorj1988** [comment]
Hi All,

Weve now shipped `0.3.2` with 3.12 support.

Thanks,
- George
